README.cs.md

```diff
@@ -0,0 +1,95 @@
+# sway
+
+[English][en] - **[Česky][cs]** - [Deutsch][de] - [Dansk][dk] - [Español][es] - [Français][fr] - [Svenska][sv] - [Ελληνικά][gr] - [हिन्दी][hi] - [Magyar][hu] - [فارسی][ir] - [Italiano][it] - [日本語][ja] - [한국어][ko] - [Nederlands][nl] - [Polski][pl] - [Português][pt] - [Română][ro] - [Русский][ru] - [Türkçe][tr] - [Українська][uk] - [中文-简体][zh-CN] - [中文-繁體][zh-TW]
+
+sway je s [i3] kompatibilní [Wayland] kompozitor. Přečtěte si [FAQ]. Připojte se na
+[IRC kanál] \(#sway na irc.libera.chat).
+
+## Podpisy vydání
+
+Vydání jsou podepsána [E88F5E48] a publikována [na GitHubu][GitHub releases].
+
+## Instalace
+
+### Z balíčků
+
+Sway je dostupný ve spoustě distribucí. Zkuste nainstalovat balíček "sway" ve vaší
+distribuci.
+
+### Kompilace ze zdrojových kódů
+
+Podívejte se na [tuto stránku wiki][Development setup], pokud chcete sestavit HEAD
+sway a wlroots pro testování nebo vývoj.
+
+Nainstalujte závislosti:
+
+* meson \*
+* [wlroots]
+* wayland
+* wayland-protocols \*
+* pcre2
+* json-c
+* pango
+* cairo
+* gdk-pixbuf2 (volitelné: oznamovací oblast)
+* [scdoc] (volitelné: manuálové stránky) \*
+* git (volitelné: informace o verzi) \*
+
+_\* Závislost pouze pro sestavení_
+
+Spusťte tyto příkazy:
+
+    meson build/
+    ninja -C build/
+    sudo ninja -C build/ install
+
+Na systémech bez logind či seatd musíte binární soubor sway nastavit jako suid:
+
+    sudo chmod a+s /usr/local/bin/sway
+
+Sway svá root oprávnění zruší krátce po spuštění.
+
+## Konfigurace
+
+Pokud již používáte i3, zkopírujte svou konfiguraci i3 do `~/.config/sway/config`
+a ta bude ihned fungovat. Jinak zkopírujte do `~/.config/sway/config` ukázkový
+konfigurační soubor. Ten se obvykle nachází v `/etc/sway/config`.
+Pro více informací o konfiguraci spusťte `man 5 sway`.
+
+## Spuštění
+
+Spusťte `sway` z TTY. Některé správce zobrazení mohou fungovat, ale nejsou
```

Comment:
```suggestion
Spusťte `sway` z TTY. Někteří správci zobrazení mohou fungovat, ale nejsou
```

Reply:
Hi Ondřej, I believe that "Některé správce" is actually correct since the display manager is a non-living thing, not a living being inside the computer managing your display. :-)

Reply:
Ah, the intricate parts of the language :)